SingularityUI/app/actions/api/webhooks.es6

```diff
@@ -2,7 +2,7 @@ import { buildApiAction, buildJsonApiAction } from './base';
 
 export const FetchWebhooks = buildApiAction(
   'FETCH_WEBHOOKS',
-  {url: '/webhooks'}
```

Comment:
This was also a bug, and will need to be fixed independently if this PR gets rejected.